examples/megaclc/megaclc.cpp

```diff
@@ -1159,11 +1159,13 @@ void reportMessageHuman(c::MegaChatHandle chatid, c::MegaChatMessage *msg, const
         const char* firstname_ptr = nullptr;
         if (room)
         {
-            firstname_ptr = room->getPeerFirstnameByHandle(handle);
+            firstname_ptr = g_chatApi->getUserFirstnameFromCache(handle);
         }
         if (firstname_ptr && *firstname_ptr != '\0')
         {
-            return std::string{firstname_ptr};
+            std::string name(firstname_ptr);
+            delete [] firstname_ptr;
+            return name;
```

Comment:
Simply return `firstname_ptr->get()` and it will implicitly create the `string` from the `const char *`. Remember it's a C-string ;)